dev_scripts_helpers/thin_client/setenv.sh

```diff
@@ -97,6 +97,12 @@ umask 002
 # modifications before starting to develop.
 set_symlink_permissions .
 
+#################################################################################
+# Install pre-commit hooks.
+#################################################################################
+
+$HELPERS_ROOT_DIR/dev_scripts_helpers/git/git_hooks/install_hooks.py --action install
```

Comment:
How can we control this to release and allow rollbacks in case there are problems?

We could have maybe a switch in repo_config to say for which repos we enforce this and which not (although probably we want to enable it everywhere, including Tutorial)

In general we need a way to:
- release something (like the hooks)
- let a subset of people try it and report feedback
- release to everyone
- revert if there are problems

We should have a way to formalize this process of releasing for devs and for prod (like feature flags)